### ISSUE TITLE
Trsnflow

### ISSUE THREAD

**Issue by simplysameer333** [simplysameer333/cordapp-tutorial#43]
Install python 6.3.6 64 bit 
add Phthon root and script dir in PATH env
 

set http_proxy=http://applicationwebproxy.nomura.com:80
set https_proxy=https://applicationwebproxy.nomura.com:80

python -m pip install --upgrade pip 
pip3 --proxy http://applicationwebproxy.nomura.com:80 install tensorflow
pip3 --proxy http://applicationwebproxy.nomura.com:80 install pandas
pip3 --proxy http://applicationwebproxy.nomura.com:80 install matplotlib
pip3 --proxy http://applicationwebproxy.nomura.com:80 install sklearn
pip3 --proxy http://applicationwebproxy.nomura.com:80 install pydotplus
pip3 --proxy http://applicationwebproxy.nomura.com:80 install scipy
pip3 --proxy http://applicationwebproxy.nomura.com:80 install graphviz 

FOr Graph - C:\Temp\python_files\graph>tensorboard --logdir .

optimizers working - they calculate the loss and keep varibale updating in correct direction
Gradient Decent Optimizer
1) (square of Sum all the differnece btween target out put and actula output )/2  ===> error calculation
2) change in varible = learning rate / rate of chnage of loss   => calucate rate on change
3) new vaible = old varible  + delta in second step =>update the varible to delta

Keep doing it unless loss in min.

Use Case Mine identifier -> STEPS
1) process data set (read)
2) Define features and labels (rock or Mine)
3) Encode Dependent varibles - label
4) Divide data in 2 parts - training and testing
5) Use Tensor flow data sets
6) Write model (login)
7) Train model
8) Calulutate error
9) Test Model on data
